:100755 100755 5fe87f0 0000000 M	debian/rules

diff --git a/debian/rules b/debian/rules
index 5fe87f0..050bfd4 100755
--- a/debian/rules
+++ b/debian/rules
@@ -20,2 +20,4 @@ override_dh_auto_configure:
 	dh_auto_configure --verbose --builddirectory=build -- \
+		--disable-doc --enable-pubkey-ciphers="dsa elgamal rsa ecc" --enable-digests="crc gostr3411-94 rmd160 sha256 sha512 sha3 tiger whirlpool stribog blake2" \
+		--with-fips-module-version=SAP SE Garden Linux nightly Libgcrypt Cryptographic Module \
 		--enable-noexecstack \
@@ -62,9 +64,2 @@ override_dh_auto_install-arch:
 
-override_dh_installinfo:
-	dh_installinfo
-	if test -e debian/libgcrypt20-doc ; then \
-		cd debian/libgcrypt20-doc/usr/share/info && \
-		sed -i -e 's:image src="\([^"]*.png"\):image src="/usr/share/doc/libgcrypt20-doc/html/\1:g' *.info* ; \
-	fi
-
 %:
